src/python/zquantum/core/evolution_test.py

```diff
@@ -109,7 +109,7 @@ def test_time_evolution_derivatives(self):
         )
         time_symbol = sympy.Symbol("t")
         time_value = 0.4
-        symbols_map = [(time_symbol, time_value)]
+        [(time_symbol, time_value)]
```

Comment:
Looks like the logic in this test needs some fixing, as `time_value` is not actually needed.

Reply:
Let's do the refactor in another PR